dictionary.txt

```diff
@@ -0,0 +1,198 @@
+Deutsch
+Jozsa
+Abhinav
+Aer
+Akira
+Andriyash
+Aroosa
+Arunachalam
+Babbush
+Barowski
+Bengio
+Bergholm
+Bertels
+Bertet
+Boeblingen
+Boixo
+Boulant
+Briegel
+Broughton
+Brukner
+Bucher
+Carleo
+Carrera
+Cerezo
+Chuang
+Cincio
+Colbeck
+Corcoles
+Courville
+Derks
+Deutschle
+Dewes
+Dunjko
+Elies
+Esteve
+Faehrmann
+Farhi
+Fock
+Francesco
+Fredkin
+Gacon
+Garey
+Glick
+Goemans
+Gogolin
+Goodfellow
+Greenberger
+Griffiths
+Gutmann
+Hadamard
+Haide
+Hamiltonians
```

Comment:
Should there also be a singular version of this?

Reply:
Done!